app/controllers/hq/partners_controller.rb

```diff
@@ -6,6 +6,19 @@ def index
     @partners= Partner.all
   end
 
+  def new
+    @partner = Partner.new
+    load_associations
+  end
+  
+  def create 
+    build_partner
+    unless save_partner
```

Comment:
@PurityControl Good idea.

@azangru, @razorcd: Any objections to making `create` & `update` more alike?